CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.0.0-rc.3 (2018-11-30)
+
+- Fixed select field components to use the appropriate CSS style when control value is invalid. [#3](https://github.com/blackbaud/skyux-select-field/pull/3)
```

Comment:
"component" should be singular since we are fixing the source, not individual instances.

I'd either add "the" before "select field" and before "control value," or I'd delete "the" before "appropriate." We shouldn't alternate between using articles and not using them, especially not within a single sentence. (I've made a note to follow up on the team to determine a style on this for changelogs. Previously, we had one style for the skyux2 repo and another style for the builder repo, but now that we have so many repos, we probably need to agree to one pattern to follow.)

Should "style" be "styles"? Not sure if it's a single CSS style or multiple styles that handle this. The PR says "stylings," so just want to make sure. Thanks.

Reply:
Updated based on this feedback.